dpsg/docs/integrations/adyen/config.md

```diff
@@ -54,7 +54,7 @@ From their you can set up `Standard Notification` webhook.
 ### Webhook Config
 
 **Transport**
-- URL: https://dpsg.`{region}`.deity.cloud/api/payment/webhook/{organization}/{project}/{environment}/adyen (replacing `{organization}`, `{project}` and `{environment}` with your details). If your region is Europe is will be omitted (https://dpsg.deity.cloud).
+- URL: https://dpsg.{region}.deity.cloud/api/payment/webhook/{organization}/{project}/{environment}/adyen (replacing `{organization}`, `{project}` and `{environment}` with your details). If your region is Europe is will be omitted (https://dpsg.deity.cloud).
```

Comment:
Typo and it should not be `will` but `should`

`If your region is Europe, the region should be omitted from the url (https://dpsg.deity.cloud).`

Reply:
@jonas-at-deity - i'm imagining customers getting the URL from the env config, as in, they will already have the URL. so  in that sense `will` makes sense.  I'm happy either way though if you feel strongly about this?